packages/calendar/src/utils/dayTemplate.js

```diff
@@ -14,14 +14,50 @@ const defaultMonthLabels = [
   'November',
   'December',
 ];
+const firstWeekDays = [1, 2, 3, 4, 5, 6, 7];
+const lastDaysOfYear = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
 
 export function dayTemplate(day, { weekdays, monthsLabels = defaultMonthLabels } = {}) {
   const dayNumber = day.date.getDate();
   const monthName = monthsLabels[day.date.getMonth()];
   const year = day.date.getFullYear();
   const weekdayName = weekdays[day.weekOrder];
+
+  let firstDay = false;
+  let endOfFirstWeek = false;
+  let startOfFirstFullWeek = false;
+  let endOfLastFullWeek = false;
+  let startOfLastWeek = false;
+  let lastDay = false;
+
+  firstDay = dayNumber === 1;
+  endOfFirstWeek = day.weekOrder === 6 && firstWeekDays.includes(dayNumber);
```

Comment:
Maybe just define them as consts here directly (without giving boolean values to what ends up to be numbers)